TCAT/Controllers/FavoritesTableViewController.swift

```diff
@@ -10,6 +10,11 @@ import UIKit
 import DZNEmptyDataSet
 import FutureNova
 
+protocol FavoritesSelectionDelegate: class {
+    /** Indicates to `HomeMapViewController` that it should reload its table. */
```

Comment:
hm, should we be commenting like this for single line comments? @kevinchan159

Reply:
Shouldn't this be `//`? `///` are typically used for docstrings, no?

Reply:
@young-k I had intended that as a docstring so that others on the team could figure out what that function's purpose is without navigating to the file. That might make more sense as a comment though. I'll change it if you think a comment is more appropriate.